alembic/env.py

```diff
@@ -15,11 +15,17 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
+<<<<<<< HEAD
+import os
+import sys
+sys.path.append(os.getcwd())
+=======
```

Comment:
I think you need to resolve these merge conflicts.

Reply:
Thanks, I will do that immediately.